Templates/TimerTrigger-FSharp/readme.md

```diff
@@ -0,0 +1,11 @@
+# TimerTrigger - F<span>#</span>
+
+The `TimerTrigger` makes it incredibly easy to have your functions executed on a schedule. This sample demonstrates a simple use case of calling your function every 5 minutes.
```

Comment:
This should be every minute, based on the CRON Expression. Looks like C# and node js template also has this text incorrect